src/frontend/CMakeLists.txt

```diff
@@ -25,4 +25,4 @@ add_executable(openbangla-gui ${SRC_MAIN} ${SRC_GUI})
 target_link_libraries(openbangla-gui libShared 3rdParty Qt5::Widgets Qt5::Network ${ZSTD_LIBRARIES})
 
 install(TARGETS openbangla-gui
-        RUNTIME DESTINATION ${PROJECT_DATADIR})
+        RUNTIME DESTINATION ${LIBEXECDIR})
```

Comment:
I'd like to place the `openbangla-gui` binary under the `/usr/lib/openbangla-keyboard` directory. It seems like we'll get a fcitx backend #202 in the future, so let's follow the `mozc` :grin: